tests/benchdnn/eltwise/eltwise.cpp

```diff
@@ -354,7 +354,11 @@ void setup_cmp(compare::compare_t &cmp, const prb_t *prb, data_kind_t kind,
                 const auto &source
                         = ((prb->dir & FLAG_BWD) && prb->use_dst()) ? dst : src;
                 const float s = source.get_elem(args.idx);
-                if (check_abs_err(prb, s, args.trh))
+                if ((check_abs_err(prb, s, args.trh) || (prb->alg == alg_t::ELU)
+                            || (prb->alg == alg_t::LOGISTIC)
+                            || (prb->alg == alg_t::SRELU))
+                        && (prb->dir & FLAG_FWD) && (prb->dt == dnnl_f16)
+                        && (is_amd_gpu()))
```

Comment:
The description misses the problem statement. Please post the output of benchdnn for provided algorithms in case of errors in the first place.

Reply:
I added some additional info in the description. Please if I miss something more let me know. Thanks.

Reply:
Just looking on the logistic output, it seems it's better to tighten the condition and consolidate the logic. Please refer to the [pooling cuda approach](https://github.com/oneapi-src/oneDNN/blob/main/tests/benchdnn/pool/pool.cpp#L163).

The suggestion is to introduce identical amd function for eltwise, check entry conditions as it is done now, and use `args.diff < epsilon_dt(args.dt)` as a resolution. For some reason the output differs by 1 ulp while it shouldn't. I assume MIOpen doesn't do division in f32 as it should.

Reply:
I considered the feedback and included a new function as the suggested one. Thanks.